napari/_tests/test_viewer.py

```diff
@@ -166,16 +166,21 @@ def test_screenshot(make_napari_viewer):
 
 def test_changing_theme(make_napari_viewer):
     """Test changing the theme updates the full window."""
-    viewer = make_napari_viewer()
+    viewer = make_napari_viewer(show=False)
+    viewer.window.qt_viewer.set_welcome_visible(False)
     viewer.add_points(data=None)
-    assert viewer.theme == 'dark'
+    size = viewer.window.qt_viewer.size()
+    viewer.window.qt_viewer.setFixedSize(size)
 
+    assert viewer.theme == 'dark'
     screenshot_dark = viewer.screenshot(canvas_only=False)
 
     viewer.theme = 'light'
     assert viewer.theme == 'light'
+    screenshot_light = viewer.screenshot(canvas_only=False)
```

Comment:
Looks like this is a duplicate line of the one below

Reply:
Oh.. nice catch! :)

Reply:
Fixed!